cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -13,14 +13,15 @@
 # =============================================================================
 
 # This function finds nvcomp and sets any additional necessary environment variables.
-function(find_and_configure_nvcomp VERSION)
-
-  # Find or install nvcomp
+function(find_and_configure_nvcomp VERSION_MIN VERSION_MAX)
+  # Search for latest version of nvComp
+  rapids_find_package(nvcomp ${VERSION_MAX} QUIET)
+  # If latest isn't found, fall back to building oldest support from source
   rapids_cpm_find(
-    nvcomp ${VERSION}
+    nvcomp ${VERSION_MIN}
     GLOBAL_TARGETS nvcomp::nvcomp
     CPM_ARGS GITHUB_REPOSITORY NVIDIA/nvcomp
-    GIT_TAG c435afaf4ba8a8d12f379d688effcb185886cec1
+    GIT_TAG a6e4e64a177e07cd2e5c8c5e07bb66ffefceae84
```

Comment:
Why specify both a version and a git tag? Isn't that redundant? If not, couldn't that lead to conflicts?

Reply:
Good point I can use the 2.2 git tag instead of the direct SHA1.
We do need both a version and a git tag so that we can do a search for a local version of 2.2 before falling back to building from source.